src/components/Page/components/Header/components/ActionGroup/ActionGroup.tsx

```diff
@@ -20,11 +17,9 @@ class ActionGroup extends React.Component<Props, never> {
   render() {
     const {actions, details, title, icon, active} = this.props;
 
-    const detailsMarkup = details ? (
+    const detailsMarkup = details && (
```

Comment:
Why this change?

Reply:
Wasn't necessary just came past is and decided to clean it up. 😄

Reply:
I'm not clear on when we use `&&`. Wouldn't this need to be null if it doesn't pass the condition, since it's markup?

Reply:
[Its explained more in-depth in React's docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) but the short answer is that `undefined` is part of type `ReactNode`. 😄